src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1374,36 +1375,42 @@ public boolean createOrUpdateShortcircuitParameters(StudyEntity studyEntity, Str
          * +-----------------------+----------------+-----------------------------------------+
          */
         boolean userProfileIssue = false;
-        UUID existingShortcircuitParametersUuid = studyEntity.getShortCircuitParametersUuid();
-        UserProfileInfos userProfileInfos = parameters == null ? userAdminService.getUserProfile(userId) : null;
-        if (parameters == null && userProfileInfos.getShortcircuitParameterId() != null) {
-            // reset case, with existing profile, having default short circuit params
-            try {
-                UUID shortcircuitParametersFromProfileUuid = shortCircuitService.duplicateParameters(userProfileInfos.getShortcircuitParameterId());
-                studyEntity.setShortCircuitParametersUuid(shortcircuitParametersFromProfileUuid);
-                removeShortcircuitParameters(existingShortcircuitParametersUuid);
-                return userProfileIssue;
-            } catch (Exception e) {
-                userProfileIssue = true;
-                LOGGER.error(String.format("Could not duplicate short circuit parameters with id '%s' from user/profile '%s/%s'. Using default parameters",
-                    userProfileInfos.getShortcircuitParameterId(), userId, userProfileInfos.getName()), e);
-                // in case of duplication error (ex: wrong/dangling uuid in the profile), move on with default params below
+
+        UUID studySCParametersId = studyEntity.getShortCircuitParametersUuid();
+        String scParametersToUse = scParameters;
+
+        // When no parameters given try to use user's profile parameters
+        if (scParametersToUse == null) {
+            UserProfileInfos userProfileinfos = userAdminService.getUserProfile(userId);
+            UUID profileSParamId = userProfileinfos != null ? userProfileinfos.getShortcircuitParameterId() : null;
+
+            if (profileSParamId != null) {
+                try {
+                    scParametersToUse = shortCircuitService.getParameters(profileSParamId);
+                } catch (Exception e) {
+                    userProfileIssue = true;
+                    LOGGER.error(String.format("Could not retrieve short circuit parameters with id '%s' from user/profile '%s/%s'. Using default parameters",
+                            profileSParamId, userId, userProfileinfos.getName()), e);
+                    // In case of error (ex: wrong/dangling uuid in the profile), fall back to default parameters
+                    scParametersToUse = scParameters;
```

Comment:
can be removed, same as line 1380
And maybe the comment can be remobed too, cause the LOG says everything ?